StreamVideo.xcodeproj/project.pbxproj

```diff
@@ -3289,9 +3289,10 @@
 				CLANG_CXX_LANGUAGE_STANDARD = "gnu++20";
 				CODE_SIGN_IDENTITY = "Apple Development";
 				CODE_SIGN_STYLE = Automatic;
-				CURRENT_PROJECT_VERSION = 15;
+				CURRENT_PROJECT_VERSION = 1;
 				DEVELOPMENT_TEAM = EHV7XZLAHA;
-				GENERATE_INFOPLIST_FILE = YES;
+				GENERATE_INFOPLIST_FILE = NO;
+				GENERATE_PKGINFO_FILE = YES;
```

Comment:
what's this and why do we need to change it?

Reply:
I transfered these options from Chat repo. Without them the generated `.ipa` file used the wrong `Info.plist` which was breaking the TestFlight uploading process.